packages/terra-infinite-list/src/_InfiniteUtils.js

```diff
@@ -0,0 +1,163 @@
+const getTopFromTopDown = (scrollItems, index, validTop) => {
```

Comment:
Adding some JSDoc style comments to these util functions would go a long way to make them easier to read through and understand.

Reply:
Updated.